vue2-vuetify/src/controls/DateControlRenderer.vue

```diff
@@ -155,9 +158,19 @@ const controlRenderer = defineComponent({
         : true;
     },
     pickerIcon(): string {
-      return typeof this.appliedOptions.pickerIcon == 'string'
-        ? this.appliedOptions.pickerIcon
-        : 'mdi-calendar';
+      if (typeof this.appliedOptions.pickerIcon == 'string') {
+        return this.appliedOptions.pickerIcon;
+      }
+
+      if (this.pickerType == 'year') {
+        return 'mdi-alpha-y-box-outline';
+      }
+
+      if (this.pickerType == 'month') {
```

Comment:
Please use strict comparisons to avoid unexpected bugs :)
```suggestion
      if (this.pickerType === 'month') {
```